.gitignore

```diff
@@ -60,7 +60,7 @@ coverage.xml
 *.pot
 
 # Sphinx documentation
-docs/_build/
+doc/source/_build
```

Comment:
I think this is covered by the `build/` entry up above. The docs get built in `doc/build` per the `Makefile`/`make.bat`. There's even another entry to ignore the docs build on line 24.